### PR TITLE
Add support for Gaussian policies in addition to categorical distributions

### PR DESCRIPTION
**Pull request by mjacar**
Very much a WIP currently

### CODE REVIEW

**Comment by mjacar** [comment]
Won't do. I'd advise anyone wanting to do this to check out [Ilya Kostrikov's repo](https://github.com/ikostrikov/pytorch-trpo).

The TLDR on how to do this is to: 
1. Edit the policy model to output the mean and the standard deviation of a Gaussian instead of a softmax.
2. When the time comes to choose an action, use the mean and std from the policy model output to sample from a Gaussian. 
3. When calculating the KL divergence, use [the formula for the KL divergence between Gaussians](https://stats.stackexchange.com/questions/7440/kl-divergence-between-two-univariate-gaussians).
